Build fails when -DWITH_OPENBABEL=OFF is specified

If cmake is configured like this:
```
x86_64-w64-mingw32-cmake -DWITH_PYTHON=OFF -DWITH_OPENBABEL=OFF ..
```

The `make` fails with:

> In file included from /home/devel/pteros/src/core/io/pdbqt_file.h:30,
> from /home/devel/pteros/src/core/io/mol_file.cpp:38:
> /home/devel/pteros/src/core/io/babel_wrapper.h:35:10: fatal error: openbabel/obconversion.h: No such file or directory
>  #include <openbabel/obconversion.h>

Now I get another error:

> In file included from /home/devel/pteros/src/core/io/gromacs_trajectory_file.cpp:32:
> /home/devel/pteros/include/pteros/core/logging.h: At global scope:
> /home/devel/pteros/include/pteros/core/logging.h:49:34: error: 'stdout_sink_st' is not a member of 'spdlog::sinks'
>    std::shared_ptr<spdlog::sinks::stdout_sink_st> console_sink;

Here is the way to reproduce it. Start a container: `sudo docker run -it burningdaylight/docker-mingw-qt5 /bin/bash`
and then inside the container:
```
git clone https://github.com/yesint/pteros.git pteros;
cd pteros; rm -rf build; mkdir build; cd build
x86_64-w64-mingw32-cmake -DWITH_PYTHON=OFF -DWITH_OPENBABEL=OFF ..
make
```

--- a/include/pteros/core/logging.h
+++ b/include/pteros/core/logging.h
@@ -30,6 +30,7 @@
 
 #include "spdlog/spdlog.h"
 #include "spdlog/fmt/ostr.h"
+#include "spdlog/sinks/stdout_sinks.h"
 #include <iostream>
 
 namespace pteros {

